hessian.go

```diff
@@ -175,16 +175,15 @@ func (h *HessianCodec) ReadBody(rspObj interface{}) error {
 
 	switch h.pkgType & 0x2f {
 	case PackageResponse | PackageHeartbeat | PackageResponse_Exception, PackageResponse | PackageResponse_Exception:
-		rsp, ok := rspObj.(*Response)
-		if !ok {
-			return perrors.Errorf("@rspObj is not *Response, it is %s", reflect.TypeOf(rspObj).String())
-		}
-		rsp.Exception = ErrJavaException
 		decoder := NewDecoder(buf[:])
 		exception, err := decoder.Decode()
 		if err != nil {
 			return perrors.WithStack(err)
 		}
+		rsp, ok := rspObj.(*Response)
+		if !ok {
+			return perrors.Errorf("java exception:%s", exception.(string))
```

Comment:
Why not return type assertion error?

Reply:
For compatibility with previous versions.

Reply:
@fangyincheng ok.